src/data/feedLibrary.js

```diff
@@ -41,6 +40,7 @@ export async function loadBlogPost(ref, feedDAC) {
  * @return {object} post object data in SkyStandards format
  */
 export async function loadBlogProfile(userID, feedDAC) {
-  const postsLoader = await feedDAC.loadPostsForUser(userID);
+  const blogSkapps = ["skynetblog.hns"]; //skapps paths to load posts from
```

Comment:
Follow Up: `skynetblog.hns` is not referenced in a lot of placed. This should be pulled into a const somewhere that all the locations can reference. 

`src/data/consts.js` for example.